sputnikdao-factory2/README.md

```diff
@@ -16,9 +16,9 @@ set CONTRACT_ID "dev-1608694678554-8567049"
 near call $CONTRACT_ID new '{}' --accountId $CONTRACT_ID 
 
 # bash
-ARGS=`echo '{"purpose": "test", "council": ["testmewell.testnet", "illia"], "bond": "1000000000000000000000000", "vote_period": "1800000000000", "grace_period": "1800000000000"}' | base64`
```

Comment:
Here we can decide about `bond` value, voting period and `grace_period` but after modifications we can't. 

How we could improve `ARGS` to define them ?